### ISSUE TITLE
Finish Homework for week 1.

### ISSUE THREAD

**Issue by domanhtien2011** [domanhtien2011/ML_CoderSchool_Homework_1#1]
@tranctan96

**Comment by coderschoolreview** [comment]
## Goal of this Assignment
The goal of this assignment was to introduce you to 2 main concepts in Machine Learning: 
- Data Visualization
- Exploratory Data Analysis. 

You learn how to query and clean data using pandas library in Python, make some plots which help to understand more about data with Seaborn library.

## Things you did well:
- You go through every single question. What a great start !
- Your solution is comprehensive and your code is clean, helping me a lot when grade your work :D
- You also make use of new library `IPython.display` there to print our solutions beautifully. Awesome !

## To sum up:
- Overall, what a beautiful work! You are demonstrating that you deeply understand the material and doing a great job of applying it. Keep it up!
